andela/ah-frontend-tesseract
# #160814326 Implement rating functionality

#### What does this PR do?
Enables users to rate an article and view the average rating of an article

#### Description of Task to be completed?
- implement the functionality of rating an article
- enable users to view the average rating of an article and their previous rating

#### How should this be manually tested?
- run `npm start` or go to the review app corresponding to this PR
-  go to the view of a single article. You should be able to see the average rating of the article.
- If you're logged in, you should be able to see 5 yellow stars that will allow you to rate the article.

#### What are the relevant pivotal tracker stories?
[#160814326](https://www.pivotaltracker.com/story/show/160814326)

#### Screenshots (if appropriate)
<img width="1385" alt="screen shot 2018-10-25 at 18 19 32" src="https://user-images.githubusercontent.com/26762336/47511362-b93e0e00-d882-11e8-8505-749b795abf40.png">
<img width="1336" alt="screen shot 2018-10-25 at 18 19 59" src="https://user-images.githubusercontent.com/26762336/47511386-c3f8a300-d882-11e8-86df-6e778b1b76d4.png">
